TripKitUI/views/TKAlertInfoView.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  TKAlertInfoView.swift
+//  TripGo
+//
+//  Created by Kuan Lun Huang on 23/9/17.
+//  Copyright © 2017 SkedGo Pty Ltd. All rights reserved.
+//
+
+import UIKit
+
+import TripKit
+
+public class TKAlertInfoView: UIView {
+  
+  @IBOutlet weak var titleLabel: UILabel!
+  @IBOutlet weak var instructionLabel: UILabel!
+  
+  
+  /// Indicating whether the alert info view is currently showing the full
+  /// content or only the title.
+  private(set) var isShowingFullContent = true
+  
+  
+  /// The alert used to configure the alert info view. Specifically, the
+  /// `title` and `text` properties will be used by the titleLabel and
+  /// instructionLabel respectively.
+  ///
+  public var alert: Alert? {
```

Comment:
Please use a protocol here and then where TKAlertInfoView is used, conform the corresponding model class to the protocol. Main reason for that is that we have non-CoreData alerts, too.